### PR TITLE
include build scripts in package.json

### PR DESCRIPTION
**Pull request by andrewharvey**
similar to how it's done in https://github.com/mourner/rbush

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,8 +3,13 @@
   "version": "2.1.0",
   "description": "k-neareset neighbors search for RBush",
   "main": "index.js",
+  "jsdelivr": "rbush-knn.js",
+  "unpkg": "rbush-knn.js",
   "scripts": {
-    "test": "eslint *.js && tape test.js"
+    "test": "eslint *.js && tape test.js",
```
Comment (mourner) [inline package.json]: The build fails because ESLint is picking up the built files. Let's make this line more specific.

File: package.json [hunk 2]
```diff
@@ -30,5 +37,9 @@
   "eslintConfig": {
     "extends": "mourner"
   },
+  "files": [
+      "knn.js",
+      "knn.min.js"
```
Comment (mourner) [inline package.json]: You're building `rbush-knn.*` but `files` point to different files.

**Review by mourner — CHANGES_REQUESTED** [review]

**Comment by andrewharvey** [comment]
Whoops, I've fixed those now.

File: package.json [hunk 3]
```diff
@@ -3,8 +3,13 @@
   "version": "2.1.0",
   "description": "k-neareset neighbors search for RBush",
   "main": "index.js",
+  "jsdelivr": "rbush-knn.js",
+  "unpkg": "rbush-knn.js",
```
Comment (mourner) [inline package.json]: btw I think we should target `min` versions in CDN defaults

Comment (andrewharvey) [reply package.json]: agreed, done.

**Review by mourner — APPROVED** [review]

**Pull request merged**